x-pack/plugins/threat_intelligence/server/utils/indicator_name.ts

```diff
@@ -63,7 +62,7 @@ const fieldTypeCheck = (type: string) =>
  * Generates Painless condition checking if given `field` has value
  */
 const fieldValueCheck = (field: string) =>
-  `if (doc['${field}'].size()!=0 && doc['${field}'].value!=null)`;
+  `if (doc.containsKey('${field}') && !doc['${field}'].empty && doc['${field}'].size()!=0 && doc['${field}'].value!=null)`;
```

Comment:
following [this documentation](https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-walkthrough.html#_missing_keys) for missing keys and missing values